lib/documents/schemas/research_for_development_outputs.json

```diff
@@ -0,0 +1,987 @@
+{
+  "content_id": "af7c7cdb-1335-4f1c-b21c-b3c8bfec8a1b",
+  "base_path": "/research-for-development-outputs",
+  "format_name": "Research for Development Output",
+  "name": "Research for Development Outputs",
+  "description": "Find outputs from Research for Development projects",
+  "filter": {
+    "format": "research_for_development_output"
+  },
+  "signup_content_id": "a062704c-f49d-4942-aaf4-c06d81ada3b8",
```

Comment:
Same here as the old finder, I think it'll need to be changed.

Reply:
Does this need to relate to anything else, or just a new UUID?